new-quote-submission-code/index.js

```diff
@@ -0,0 +1,53 @@
+let express = require("express");
+let app = express();
+let bodyParser = require("body-parser");
+let http = require("http").Server(app);
+import { Client } from 'pg';
+const client = new Client();
+
+/*
+TODO: need to use the following environment variables, which are expected by the pg npm package,
+in order to successfully connect to the postgres db (source: https://node-postgres.com/features/connecting):
+
+PGUSER
+PGHOST
+PGPASSWORD
+PGDATABASE
+PGPORT
+
+(in replit, these should be secrets. otherwise, these can be environment variables)
+*/
+await client.connect();
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.get("/submit", function(req, res) {
+  res.sendFile("/submit.html", { root: "." });
+});
+
+app.post("/submit", async function(req, res) {
+  const submissionTimeDateObj = new Date(Date.now());
+  const submissionTime = submissionTimeDateObj.toTimeString();
```

Comment:
Is this the time format you want? ("hh:mm:ss UTC-offset (time_zone)")
![292564639-4f681d96-f950-46d0-975f-734b58f754fc](https://github.com/blairbasheng/quotes/assets/46426339/3e55ea27-0182-4837-b3b4-d4250a79e5ed)
The csv uses "YYYY-MM-DD hh:mm:ss"
![292564990-fa9878cb-d8cf-4497-b7b8-5ebdb29328c6](https://github.com/blairbasheng/quotes/assets/46426339/b0ecd968-7ba6-44e5-979b-ccd5a2ef5370)